clar2wasm/tests/wasm-generation/maps.rs

```diff
@@ -171,15 +171,23 @@ proptest! {
             type_string(&vty)
         );
 
+        let expected_set = Value::cons_list_unsanitized(expected_set);
+        prop_assume!(expected_set.is_ok(), "Couldn't generate a valid list");
+        let expected_set = expected_set.unwrap();
+
+        let expected_get = Value::cons_list_unsanitized(expected_get);
+        prop_assume!(expected_get.is_ok(), "Couldn't generate a valid list");
```

Comment:
Wondering if some weighted strategy would simplify this. 🤔

Reply:
Certainly, some changes to the list generation can be made. We can add that as an improvement for the proptests.

Reply:
@moodmosaic This issue has only been detected once since it exists, I don’t think we should overthink it.